tests/unit/experiment/test_utils.py

```diff
@@ -112,3 +113,26 @@ def test_divide_nested_grid_search_options_nested_options(make_classes):
     config2 = yaml.load(txt_2)
     divided_configs = list(divide_nested_grid_search_options(config))
     assert repr(divided_configs) == repr([config1, config2])
+
+
+def test_default_devices():
+    ray.init()
+    devices = get_default_devices(debug=False)
+    assert devices['cpu'] == 1
+
+    devices = get_default_devices(debug, default_cpus=2)
```

Comment:
`debug` has no value here

Reply:
forgot `debug=False`?

Reply:
I think you forgot `debug=False` as the first line of the test

Reply:
yep my bad